src/mrsrequest/forms.py

```diff
@@ -33,6 +33,37 @@
 '''
 
 
+# Custom form field allowing decimal inputs with commas instead of dots
+# Quite a useful method in France, since we mainly use commas for decimal
+# numbers
+class AllowedCommaDecimalField(forms.DecimalField):
+
+    # We override the to_python method to look for a comma inside the input
+    # value and if found, to replace it with a standard decimal dot before
+    # passing the value to the standard DecimalField.
+    def to_python(self, value):
+
+        comma_index = 0
+
+        if value:
+
+            # First we check if there is a comma inside the decimal input, and
+            # store its position
+            try:
+                comma_index = value.index(',')
+            except ValueError:
+                pass
+
+            # If we found a comma inside the input value, we replace it with a
+            # regular dot
+            if comma_index:
+                value = value.replace(',', '.')
+
+        # We finally pass the value (modified or not) to the standard
+        # DecimalField
+        return super(AllowedCommaDecimalField, self).to_python(value)
```

Comment:
Est-ce-que cette solution fonctionnait sinon ? https://docs.djangoproject.com/en/dev/topics/forms/modelforms/#enabling-localization-of-fields

Reply:
Je ne connaissais pas cette solution, mais non elle ne fonctionne pas, malheureusement (elle aurait été plus simple et propre effectivement).

Reply:
Et celle ci ?
https://docs.djangoproject.com/en/2.2/ref/settings/#decimal-separator

Reply:
J'ai du mal a croire qu'il faille autant de code pour accepter une virgule en fait ...
Est-ce-qu'on peut essayer avec decimal-separator stp ?

Reply:
Je viens de tester avec decimal-separator, et ça ne change rien au problème.

Reply:
(Sorry to be late to the party -- I noticed this commit and was curious to reproduce the problem).

I think the `base.html` template is missing the language attribute:
```suggestion
        <html lang="{{ LANGUAGE_CODE }}">
```
This will enable the browser to adapt its field validation to allow different decimal separators if required, and Django will then parse the returned value appropriately. (I think the browser maybe switches behaviour from `<input type="number">` to `<input type="text">` when lang != 'en').

It works for me when I add a `DecimalField()` to the CRUDLFAP+ example models and update `crudlfap/jinja2/crudlfap/base.html`. I can also then edit the browser HTML in Web Developer mode to reproduce the original problem (i.e. comma in Django view display format, but fullstop in browser field input format).